brian2/synapses/synapses.py

```diff
@@ -993,12 +994,12 @@ def _create_variables(self, equations, user_dtype=None):
                 self.variables.add_reference(name + '_pre', self.source, name,
                                              index=index)
             except TypeError:
-                logger.debug(('Cannot include a reference to {var} in '
-                              '{synapses}, {var} uses a non-standard indexing '
-                              'in the pre-synaptic group '
-                              '{source}.').format(var=name,
-                                                  synapses=self.name,
-                                                  source=self.source.name))
+                logger.diagnostic(('Cannot include a reference to {var} in '
+                                   '{synapses}, {var} uses a non-standard indexing '
+                                   'in the pre-synaptic group '
+                                   '{source}.').format(var=name,
+                                                       synapses=self.name,
+                                                       source=self.source.name))
```

Comment:
I see. And we don't want to display this message at a higher level because most of the time the user isn't trying to access this variable so it would just be noise. I think that's a fair compromise for now.